src/app/public/src/modules/edit-button/edit-button.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="stache-edit-button" *ngIf="url">
-  <a class="stache-btn-edit" [href]="url">
+  <a class="stache-btn-edit" [href]="url" target="_blank">
```

Comment:
Our general rule of thumb is that all links should open within the same tab, this allows users to right click and open in a new tab without forcing the rule on everyone.  Lots of debate around this, however because that is our default mode, we do have the option of opening in a new tab, but need to add both the `fa fa-external-link` icon after the text here, and `rel="noopener noreferrer"` to links containing target="_blank" as a precaution against reverse tabnabbing